pages/common/iperf3.md

```diff
@@ -0,0 +1,23 @@
+# iperf3
+
+> Iperf3 — traffic generator for testing network bandwidth
```

Comment:
Not sure `Iperf3` is needed here. Perhaps just `A traffic generator for testing network bandwidth.`?

Also, could you add a full stop to the end (as noted by the linter).